### PR TITLE
N36: Blackjack

### PR DESCRIPTION
**Pull request by setzer22**
_Part of #1098_

### CODE REVIEW

**Comment by setzer22** [comment]
The links seem to be broken on the Github preview, but I remember this format being used in past issues. Can you confirm I didn't mess up? :smile:

File: content/news/036/index.md [hunk 1]
```diff
@@ -74,6 +74,37 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+- A better data model for meshes, based on groups and channels.
+- Game engine integration with Godot, more engines coming soon.
+- Introduce Lua as an extension language.
+- Add *many* new nodes: Extrude along curve, Copy to points...
+- Add experimental support for L-Systems.
+- Reworked Look & Feel
+
+A talk about Blackjack's vision and a tour of its features was shared at the
+start of July in the [Rust gamedev meetup](blackjack-talk-yt). Interested
+developers are encouraged to [check the project out on GitHub](Blackjack) and
```
Comment (17cupsofcoffee) [inline content/news/036/index.md:102]:
```suggestion
start of July in the [Rust gamedev meetup][blackjack-talk-yt]. Interested
developers are encouraged to [check the project out on GitHub][Blackjack] and
```

**Review by 17cupsofcoffee — APPROVED** [review]
Footnote-style links use `[]` rather than `()` - I've been caught out by that before too :)

Other than that, looks fine!

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Tnx!

**Pull request merged**